contrib/stable/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# stable/Dockerfile
+#
+# Build a Skopeo container image from the latest
+# stable version of Skopeo on the Fedoras Updates System.
+# https://bodhi.fedoraproject.org/updates/?search=skopeo
+# This image can be used to create a secured container
+# that runs safely with privileges within the container.
+#
+FROM registry.fedoraproject.org/fedora:latest
+
+# Don't include container-selinux and remove
+# directories used by yum that are just taking
+# up space.
+RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install skopeo fuse-overlayfs --exclude container-selinux; rm -rf /var/cache /var/log/dnf* /var/log/yum.*;
+
+ADD https://raw.githubusercontent.com/containers/skopeo/master/contrib/skopeoimages/stable/containers.conf /etc/containers/
+
+# Adjust storage.conf to enable Fuse storage.
+RUN chmod 644 /etc/containers/containers.conf; sed -i -e 's|^#mount_program|mount_program|g' -e '/additionalimage.*/a "/var/lib/shared",' -e 's|^mountopt[[:space:]]*=.*$|mountopt = "nodev,fsync=0"|g' /etc/containers/storage.conf
+RUN mkdir -p /var/lib/shared/overlay-images /var/lib/shared/overlay-layers; touch /var/lib/shared/overlay-images/images.lock; touch /var/lib/shared/overlay-layers/layers.lock
```

Comment:
Are /var/lib/shared/overlay-images really a concern for Skopeo? How would one use that? A container running inside the Skopeo. container? Or is the one running this container expected to add `-v` to overlay over this directory?

Reply:
I'm not sure, I lifted this part from podman's Dockerfile.  Happy to remove it if you think it's not useful.

Reply:
I really have no idea.

Reply:
@rhatdan I think this line came from your noggin for Podman, needed here?

Reply:
Yes no need for this in skopeo.  This is just to allow containers to use additional stores.  I don't see a real use case for skopeo and additional stores.

podman run -v /var/lib/containers/storage:/var/lib/shared:ro quay.io:skopoe/stable skopeo copy containers-storage ...

But not really likely

Reply:
But then again no real reason not to do it, since it will not hurt it, then buildah, podman and skopeo would all have the same feature to allow their containers to run with addition-stores.

Reply:
I've left it in, but have added a comment.